src/modules/cp/components/ApproveRequestBlade/ApproveRequestBlade.const.ts

```diff
@@ -0,0 +1,25 @@
+import { format } from 'date-fns';
+import { i18n } from 'next-i18next';
+import { date, object, SchemaOf, string } from 'yup';
+
+import { getLocaleFromi18nLanguage } from '@shared/utils';
+
+import { ApproveRequestFormState } from './ApproveRequestBlade.types';
+
+export const APPROVE_REQUEST_FORM_SCHEMA = (): SchemaOf<ApproveRequestFormState> => {
+	return object({
+		accessFrom: date().required(i18n?.t('Een startdatum is verplicht.')),
+		accessTo: date().required(i18n?.t('Een einddatum is verplicht.')),
+		accessRemark: string().optional(),
+	});
+};
+
+export const ApproveRequestAccessDateFormatter = (date?: Date): string => {
+	const locale = getLocaleFromi18nLanguage(i18n?.language || '');
+	return date ? format(date, 'P', { locale }) : '';
+};
+
+export const ApproveRequestAccessTimeFormatter = (date?: Date): string => {
```

Comment:
Idem, je kan dit zelfs in een generieke functie gieten waarbij je uw format als parameter meegeeft in `shared/utils/date.ts`:
```typescript
export const formatWithLocale = (format: string, date?: Date): string => {
    const locale = getLocaleFromi18nLanguage(i18n?.language || '');
    return date ? format(date, format, { locale }) : '';
}
```